Dockerfile

```diff
@@ -0,0 +1,33 @@
+# BUILD IMAGE --------------------------------------------------------
+FROM nimlang/nim:1.2.0-alpine AS nim-build
+
+# Get build tools and required header files
+RUN apk add --no-cache bash build-base pcre-dev linux-headers
+
+RUN mkdir /app
+WORKDIR /app
+ADD . .
+
+# Ran separately from 'make' to avoid re-doing
+RUN git submodule update --init --recursive
+
+# Build the actual node binary
+RUN make wakunode
```

Comment:
Also, I believe part of Nim compilation does benefit here from `-j$(nproc)`

Reply:
Sure.